core/src/main/java/org/geysermc/geyser/registry/populator/CustomBlockRegistryPopulator.java

```diff
@@ -34,13 +34,18 @@
 import org.geysermc.geyser.util.MathUtils;
 
 import java.util.ArrayList;
+import java.util.concurrent.atomic.AtomicInteger;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
 public class CustomBlockRegistryPopulator {
+
+    // Since 1.20.60, custom blocks need a block_id in their nbt tag
+    public static AtomicInteger BLOCK_ID = new AtomicInteger(10000);
```

Comment:
since the `10000` is also used in BlockRegistryPopulator, can we make a constant for it (or any other solution to ensure longtime consistency)